e2e/integration/helpers/helpers.go

```diff
@@ -0,0 +1,341 @@
+
+}
+
+func GenerateNodeReqBody(nodeid string) (error, map[string]interface{}){
+	var temp map[string]interface{}
+
+	body:= fmt.Sprintf(`{"kind": "Node","apiVersion": "v1","metadata": {"name": "%s","labels": {"name": "edgenode"}}}`, nodeid)
+	err:= json.Unmarshal([]byte(body), &temp)
+	if err!= nil{
+		utils.Failf("Unmarshal body failed: %v", err)
+		return err, temp
+	}
+
+	return nil, temp
+}
+
+//Function to get the pods from Edged
+func DeRegisterNodeFromMaster(ctx *utils.TestContext, nodehandler, nodename string) error {
+	err, resp := SenHttpRequest(http.MethodDelete, ctx.Cfg.ApiServer+nodehandler+ "/" +nodename)
+	if err != nil{
+		utils.Failf("Sending SenHttpRequest failed: %v", err)
+		return err
+	}
+	defer resp.Body.Close()
+	Expect(resp.StatusCode).Should(Equal(http.StatusOK))
+
+	return nil
+}
+
+//Function to get the pods from Edged
+func RegisterNodeToMaster(ctx *utils.TestContext, UID, nodehandler string) error {
+	err, body := GenerateNodeReqBody(UID)
+	if err != nil {
+		utils.Failf("Unmarshal body failed: %v", err)
+		return err
+	}
+
+	client := &http.Client{}
+	t := time.Now()
+	nodebody, err:= json.Marshal(body)
+	if err != nil{
+		utils.Failf("Marshal body failed: %v", err)
+		return err
+	}
+	BodyBuf := bytes.NewReader(nodebody)
+	req, err := http.NewRequest(http.MethodPost, ctx.Cfg.ApiServer+nodehandler, BodyBuf)
+	req.Header.Set("Content-Type", "application/json")
+	if err != nil {
+		utils.Failf("Frame HTTP request failed: %v", err)
+		return err
+	}
+	resp, err := client.Do(req)
+	if err != nil {
+		utils.Failf("Sending HTTP request failed: %v", err)
+		return err
+	}
+	utils.InfoV6("%s %s %v in %v", req.Method, req.URL, resp.Status, time.Now().Sub(t))
+	defer resp.Body.Close()
+
+	Expect(resp.StatusCode).Should(Equal(http.StatusCreated))
+	return nil
+}
+
+func CheckNodeReadyStatus(ctx *utils.TestContext, nodehandler, nodename string) string{
+	var node v1.Node
+	var nodestatus ="unknown"
+	err, resp := SenHttpRequest(http.MethodGet, ctx.Cfg.ApiServer+nodehandler+ "/" + nodename)
+	if err != nil{
+		utils.Failf("Sending SenHttpRequest failed: %v", err)
+		return nodestatus
+	}
+	defer resp.Body.Close()
+
+	contents, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		utils.Failf("HTTP Response reading has failed: %v", err)
+		return nodestatus
+	}
+	err = json.Unmarshal(contents, &node)
+	if err != nil {
+		utils.Failf("Unmarshal HTTP Response has failed: %v", err)
+		return nodestatus
+	}
+
+	fmt.Println(string(node.Status.Phase))
+	return string(node.Status.Phase)
+}
+
+func CheckNodeDeleteStatus(ctx *utils.TestContext, nodehandler, nodename string) int{
+	err, resp := SenHttpRequest(http.MethodGet, ctx.Cfg.ApiServer+nodehandler+ "/" + nodename)
+	if err != nil{
+		utils.Failf("Sending SenHttpRequest failed: %v", err)
+		return -1
+	}
+	defer resp.Body.Close()
+	return  resp.StatusCode
+}
+
+func PrintCombinedOutput(cmd *exec.Cmd) error{
```

Comment:
Pls add a comment with a short description above this function explaining its functionality